particles/fcc/object_link.py

```diff
@@ -0,0 +1,16 @@
+from heppy.particles.object_link import ObjectLink as BaseLink
+from pod import POD
+from heppy.papas.data.identifier import Identifier
+
+from ROOT import TLorentzVector
+from ROOT import TVector3
+
+class ObjectLink(BaseLink, POD):
```

Comment:
missing doc in this class

Reply:
Have done. However this made me realise that whilst the base ObjectLink is fully general, this one is not at all general. So I have renamed it to ParticleMCParticleLink which is what it contains. Hope that is OK.